backend/modules/transform/domain/transform.py

```diff
@@ -0,0 +1,13 @@
+import pandas as pd
+from services.services import TransformService
+
+# for now this Transformer doesn't do anything yet
+class TransformationHandler:
```

Comment:
Could you please elaborate me what is your plan for this?
This is a really important in case of ETL and I don't really get why you implemented it that way.

Reply:
The goal of this class is to Represent "Transform" part of ETL (the same way like ExtractManager does for "Extract") .
Maybe TransformationHandler is a bad name - it should some rather "TransformManager". 

We agreed, that it's not going to do anything particular yet - accept a DataFrame and return it as it is , without transformations. 

I added abstracted service layer as a placeholder for future dataframe tranformation logic. 

I'm eager to meet on a call to discuss it.

Reply:
`TransformManager` seems fine.

Sure, we can do a call.